applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -223,3 +237,45 @@ def MarkWakeTEElement(self):
                     elem.SetValue(CPFApp.KUTTA, False)
                 else: #Rest of elements touching the trailing edge but not part of the wake
                     elem.SetValue(CPFApp.WAKE, False)
+
+    def ExecuteFinalizeSolutionStep(self):
+        node_velocity_potential_above = 0
+        node_velocity_potential_below = 0
+
+        show_only_the_first_element = True
+        for elem in self.wake_model_part.Elements:
```

Comment:
I have two questions regarding this (also tagging @inigolcanalejo ):

-Would it make sense to define this in the compute_lift_process.py? So we could print there the lift that comes from the potential jump and from the pressure distribution.

-Would it be better to just use compute the jump in the trailing edge node? So no info from the wake is needed. This is what I do in the [lift_response_function.cpp](https://github.com/KratosMultiphysics/Kratos/blob/208c5ef8f16cabbd816b9252557f505e93cca46e/applications/CompressiblePotentialFlowApplication/custom_response_functions/adjoint_lift_response_function_coordinates_jump.cpp#L49). Is there any issue from just using the trailing edge node?

![image](https://user-images.githubusercontent.com/32136457/55617327-1aec9780-5794-11e9-9c26-4dfeb6954aef.png)

Reply:
Exactly, the trailing edge node gives the correct value for the lift coefficient so we just need to use only this node. Thus the for loop over the wake elements can be removed and take the value only from this node. About where to define this process, it makes sense to define it in compute_lift_process.py so we will discuss it with Inigo and move it there eventually.